packages/cloudflare/src/api/get-cloudflare-context.ts

```diff
@@ -0,0 +1,50 @@
+import "server-only";
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface CloudflareEnv {}
+}
+
+export type CloudflareContext<
+  CfProperties extends Record<string, unknown> = IncomingRequestCfProperties,
+  Context = ExecutionContext,
+> = {
+  /**
+   * object containing the worker's bindings ([docs](https://developers.cloudflare.com/workers/runtime-apis/bindings/))
```

Comment:
nit: 

- I would drop `object containing the` (same below) - I think it doesn't add value
- `worker's bindings ([docs](https://developers.cloudflare.com/workers/runtime-apis/bindings/))` -> `[worker's binding](https://developers.cloudflare.com/workers/runtime-apis/bindings/)` ?